oci/layout/oci_dest.go

```diff
@@ -217,10 +213,6 @@ func (d *ociImageDestination) PutManifest(ctx context.Context, m []byte) error {
 		return err
 	}
 
-	if d.ref.image == "" {
-		return errors.Errorf("cannot save image with empyt image.ref.name")
-	}
-
 	annotations := make(map[string]string)
 	annotations["org.opencontainers.image.ref.name"] = d.ref.image
```

Comment:
AFAICT an empty string is not a valid value for this annotation; it should not be added if `d.ref.image == ""`.

Reply:
Fixed.
Added comments too for the `archiveReference.destinationRef` and `ociReference.image`.